src/codechicken/nei/NEIClientConfig.java

```diff
@@ -56,10 +62,35 @@
     public static final File bookmarkFile = new File(configDir, "bookmarks.ini");
     public static final File handlerFile = new File(configDir, "handlers.csv");
     public static final File serialHandlersFile = new File(configDir, "serialhandlers.cfg");
+    public static final File heightHackHandlersFile = new File(configDir, "heighthackhandlers.cfg");
+    public static final File handlerOrderingFile = new File(configDir, "handlerordering.csv");
 
     // Set of handlers that need to be run in serial
-    public static HashSet<String> serialHandlers = new HashSet<>(); 
-   
+    public static HashSet<String> serialHandlers = new HashSet<>();
+
+    // Set of handler ID of handlers that need the hack in GuiRecipe.startHeightHack().
+    public static HashSet<String> heightHackHandlers = new HashSet<>();
```

Comment:
I would suggest calling it `hardcodedHeightHandlers`/`legacyHightHandlers`/`handlersWithHardcodedHeight`/`handleersWithHardcodedScale` or smth as `hack` is a bit generic

Reply:
Eh, I think I still prefer the current name. Something like `hardcodedHeightHandlers` could make it sound like our code in NEI will hard-code the handler height or something, as opposed to the handler itself having a hard-coded NEI window height internally.